pages/products/index.js

```diff
@@ -0,0 +1,119 @@
+import { SVGIcon } from "@/components/SVGIcon";
+import { useAtom } from "jotai";
+import { RESET } from "jotai/utils";
+import Image from "next/image";
+import { useRouter } from "next/router";
+import styled from "styled-components";
+
+import { initialProducts } from "../product/[code]";
+
+const StyledAllCards = styled.div`
+  margin-left: 0.4rem;
+  width: 97vw;
+  margin-top: 4rem;
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  gap: 0.5rem;
+`;
+const StyledImage = styled(Image)`
+  width: 135px;
+  height: 135px;
+  object-fit: contain;
+`;
+const StyledProductCard = styled.section`
+  witdh: 50%;
+  //   margin-left: 3%;
+  //   width: 80%;
```

Comment:
You need to remove all deactivated code before stating a PR.

Reply:
done